changelog.d/2143.breaking.rst

```diff
@@ -0,0 +1 @@
+Setuptools adopts distutils from the standard library as a top-level ``distutils`` package and no longer depends on distutils in the standard library. This new ``distutils`` package will be masked by ``distutils`` in the standard library unless that library has been stripped by a downstream package manager or gets removed in a future version. Although this change is not expected to break any use-cases, it will likely affect tool-chains that are monkey-patching distutils or relying on Setuptools' own monkey patching of distutils.
```

Comment:
A few things about the hyphenation feels a bit off to me. I would not hyphenate use-case or tool-chain (I would say "use case" and "toolchain" or "tool chain", respectively), and "monkey-patching" and "monkey patching" both appear here. I think either one is fine (I'm leaning towards "monkey-patching", I guess), but it should be consistent.

Reply:
I agree. "Use case" (n), "toolchains/tool chains" (n) and "monkey-patching distutils" (v). For the last, if you were talking about monkey patching in general it would be different, but because you're qualifying the type of patching being applied to distutils, it gets the hyphen.

(Though tbh, I don't know the formal rules here. I've just learned that trusting my gut on grammar works well for me.)

Reply:
Good suggestions, committed in a8081a19.